test/bilinear_linearization.jl

```diff
@@ -0,0 +1,339 @@
+
+using BilevelJuMP
+# using Ipopt
+# using Cbc
+# using CPLEX
+using Gurobi
```

Comment:
I do have access to Xpress and I have merged the latest master. The Xpress/CI tests are passing for this PR (but the codecov tests are not). What do you recommend to address the codecov tests? (I am unfamiliar with them, but it looks like I need more doc strings and/or tests?)

Reply:
for codecov we basically need more small tests testing different bits of the overall code. It will tell you which pieces of code are never reached, so that you can add testes that reach those lines.
Focus on your added functions.

Reply:
dont worry about docstrings now